### ISSUE TITLE
Staging Review finding: Process list eyebrow text may be missed by screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98739]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Disability Benefits Experience Team 1
**Product:** Disability Comp Application (21-526 EZ)
**Feature:** Submission Experience

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** Yes
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the confirmation page, the process list under "What to expect" includes H3 headings for each of the steps, and eyebrow text above the headings labeling the steps as Active and Pending.

Many screen reader users navigate a page by skipping heading to heading, and when they find a heading they're interested in they have their screen reader announce the content that follows the heading. Since the eyebrow text appears before the heading text, it's unlikely that a screen reader user navigating this way will hear that text.

The impact of this is pretty small since the text of the headings is straightforward. But there's still potentially a chance that some users will be confused about what's happening now and what happens later.

<details><summary>Link, screenshot or steps to recreate</summary>

![What to expect process list with two steps. The first step has eyebrow text "Active" and a heading that reads "We'll send oyu an email to confirm your submission." The second step has eyebrow text "Pending" and a heading that reads "Next we'll send you a letter to let you know we have your claim" followed by a paragraph reading "You should get this letter in about 1 week, plus mailing time, after we receive your claim."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/86117-Brian-1.png?raw=true)

</details>

## Recommended action

The easiest option here is to wrap the eyebrow text inside the H3, and then adjust any CSS styling as needed to preserve the visual appearance, something roughly like:
```
<h3>
   <span class="eyebrow">Active</span>
   <span class="heading-text">We'll send you an email to confirm your submission</span>
</h3>
```
This will be announced as "heading level 3, Active - We'll send you an email..." which should ensure screen reader users get all of the relevant context when they land on the heading.

I'll also tag this for the Design System Team as well. Since the [process list guidance](https://design.va.gov/components/process-list#accessibility-considerations) recommends using eyebrow text, the component should support it out of the box.

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** [Process list ](https://design.va.gov/components/process-list)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by pacerwow** [comment]
Design System team will fix this. No action needed.
